src/app/modules/courses/components/course-add-edit/course-add-edit.component.html

```diff
@@ -0,0 +1,88 @@
+<div class="card">
+  <form [formGroup]="form" (submit)="save()">
+    <h2 *ngIf="isNew else currentCourse">New course</h2>
+    <ng-template #currentCourse>
+      <h2>{{form.value['title'] | uppercase}}</h2>
+    </ng-template>
+    <div class="input-field">
```

Comment:
It is too much duplicates related to this block. Maybe you can move as separate component some part and reuse them

Reply:
done